docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md

```diff
@@ -0,0 +1,90 @@
+# Comparison operators
+
+| Name  | Description |
+|:----|:----:|
+| =   | Assign a value   |
+| /   | Division operator   |
+| ==   | Equal operator   |
+| !=   | Not equal operator   |
+| <   | Less than operator   |
+| <=   | Less than or equal operator   |
+| -   | Minus operator   |
+| %   | Modulo operator   |
+| +   | Addition operator   |
+| *   | Multiplication operator   |
+| -   | Change the sign of the argument   |
```

Comment:
add two lines:
|`IS NULL` |  NULL test |
| `IS NOT NULL` | not NULL test |

and add example.